website/content/docs/connect/ca/index.mdx

```diff
@@ -211,3 +211,23 @@ servers.
 Currently both Consul and Vault CA providers _do_ support cross signing. As more
 providers are added this documentation will list any that this section applies
 to.
+
+### Recovering From Expired Certificates
+If the built-in CA provider is misconfigured or unavailable, Consul service mesh requests eventually
+stop functioning due to expiration of intermediate and root certificates. To recover manually, use the
+[CLI helper](/commands/tls/ca#consul-tls-ca-create) to generate CA certificates.
+
+
+#### Example - Regenerating the built in CA
+```shell-session
+$ consul tls ca create -cluster-id test -common-name "Consul Agent CA" -days=365 -domain consul
+  ==> Saved consul-agent-ca.pem
+  ==> Saved consul-agent-ca-key.pem
+```
+The example above generates a new CA with a validity of 365 days. The cluster-id argument is specific
+to each cluster and can be looked up by examining the `TrustDomain` field in
+the [List CA Roots](/api-docs/connect/ca#list-ca-root-certificates) endpoint.
+
+The contents of the generated cert and private key files from the above step should then be used with
+the [Update CA Configuration](/api/connect/ca#update-ca-configuration) endpoint to enable using the new CA for
+any future requests.
```

Comment:
@mkeeler @kyhavlov I was wondering if we should add something  here around what users should expect after using the update CA endpoint to update their CA after regenerating. Would saying something like "After doing this, the root and intermediate certificates used for Consul service mesh will be regenerated"? Or are there more manual steps involved before everything starts working again?

Reply:
The intermediate and all leaf certs will be regenerated. Here we are providing the root cert to use instead of any auto-generated one.

If this is being done in the primary then all secondary dcs should pick up the change and also regenerate their intermediate/leaf certs.

Reply:
I added one sentence to clarify what users should expect after this step. If that looks good to you both, I;ll merge this.